community/libqmi/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=libqmi
-pkgver=1.20.0
+pkgver=1.20.2
 pkgrel=0
 pkgdesc="QMI modem protocol helper library"
 url="http://www.freedesktop.org/wiki/Software/libqmi"
```

Comment:
makes sense to modernize a bit more `url="https://www.freedesktop.org/wiki/Software/libqmi/"`

Reply:
well spotted - fixed